packages/x-data-grid/src/hooks/features/sorting/gridSortingSelector.ts

```diff
@@ -73,3 +73,17 @@ export const gridSortColumnLookupSelector = createSelectorMemoized(
     return result;
   },
 );
+
+/**
+ * @category Sorting
+ * @ignore - do not document.
+ */
+export const gridSortedRowIndexLookupSelector = createSelectorMemoized(
```

Comment:
Do we have a use-case for it in the userland? If not, how about exporting it from `@mui/x-data-grid/internals` instead?

Reply:
I pushed a commit to further support this: https://github.com/mui/mui-x/pull/15355/commits/1fa97339560b85b40b1c6461758de5ef0a9ec6a4